vision/google/cloud/vision/_gax.py

```diff
@@ -14,12 +14,55 @@
 
 """GAX Client for interacting with the Google Cloud Vision API."""
 
+from google.cloud.gapic.vision.v1 import image_annotator_client
+from google.cloud.grpc.vision.v1 import image_annotator_pb2
+
+from google.cloud._helpers import _to_bytes
+
 
 class _GAPICVisionAPI(object):
     """Vision API for interacting with the gRPC version of Vision.
 
-    :type client: :class:`~google.cloud.core.client.Client`
+    :type client: :class:`~google.cloud.vision.client.Client`
     :param client: Instance of ``Client`` object.
     """
     def __init__(self, client=None):
-        raise NotImplementedError
+        self._client = client
+        self._api = image_annotator_client.ImageAnnotatorClient()
+
+
+def _to_gapic_feature(feature):
+    """Helper function to convert a ``Feature`` to a gRPC ``Feature``.
+
+    :type feature: :class:`~google.cloud.vision.feature.Feature`
+    :param feature: Local ``Feature`` class to be converted to gRPC ``Feature``
+                    instance.
+
+    :rtype: :class:`~google.cloud.grpc.vision.v1.image_annotator_pb2.Feature`
+    :returns: gRPC ``Feature`` converted from
+              :class:`~google.cloud.vision.feature.Feature`.
+    """
+    return image_annotator_pb2.Feature(
+        type=getattr(image_annotator_pb2.Feature, feature.feature_type),
```

Comment:
Yeah, maybe I should convert the [enums](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/61d63e56599e18de6c1b8a54c33831a95c02769a/vision/google/cloud/vision/feature.py#L24) to integers like the pb in the future?